service/sequencer_service.go

```diff
@@ -2,36 +2,51 @@ package service
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	"github.com/vocdoni/vocdoni-z-sandbox/log"
 	"github.com/vocdoni/vocdoni-z-sandbox/sequencer"
 	"github.com/vocdoni/vocdoni-z-sandbox/storage"
+	"github.com/vocdoni/vocdoni-z-sandbox/types"
 	"github.com/vocdoni/vocdoni-z-sandbox/web3"
 )
 
+// StatsMonitorInterval is the interval at which process statistics are logged.
+// This can be overridden before starting the service.
+var StatsMonitorInterval = 60 * time.Second
+
 // SequencerService represents a service that handles background vote processing.
 type SequencerService struct {
 	Sequencer *sequencer.Sequencer
+	storage   *storage.Storage
 }
 
 // NewSequencer creates a new sequencer instance. It will verify new votes, aggregate them into batches,
 // and update the ongoing state with the new ones. The batchTimeWindow defines how long a batch can wait
 // until processed (either the batch becomes full of votes or the time window expires).
 func NewSequencer(stg *storage.Storage, contracts *web3.Contracts, batchTimeWindow time.Duration) *SequencerService {
-	log.Infow("creating sequencer service", "batchTimeWindow", batchTimeWindow.String())
 	s, err := sequencer.New(stg, contracts, batchTimeWindow)
 	if err != nil {
 		log.Fatalf("failed to create sequencer: %v", err)
 	}
 	return &SequencerService{
 		Sequencer: s,
+		storage:   stg,
 	}
 }
 
 // Start begins the vote processing service. It returns an error if the service is already running.
 func (ss *SequencerService) Start(ctx context.Context) error {
-	return ss.Sequencer.Start(ctx)
+	// Start the sequencer
+	if err := ss.Sequencer.Start(ctx); err != nil {
+		return err
+	}
+
+	// Start the stats monitor
+	ss.startStatsMonitor(ctx, StatsMonitorInterval)
+
+	return nil
```

Comment:
Why at `SequencerService` level instead of in the `Sequencer`? There you already have the `storage` and other references used in the stats monitor (`ActiveProcessIDs`, ...)